client/src/utils/helpers.tsx

```diff
@@ -1,23 +1,19 @@
-function apiServer() {
+export function apiServer() {
   const url = window.location.href;
   const slice = url.slice(0, 21);
-  if (slice === 'http://localhost:3000') {
-    return 'http://localhost:3001'
-  } else {
+  if (slice === "http://localhost:3000") {
+    return "http://localhost:3001";
+  } else if (slice === "https://dragon-fruit-") {
```

Comment:
Should the slice be https://dragon-fruit-app ?

Reply:
No because this has to be the first 22 characters. This is kind of a hacky way to get the API to work in both local and live environments. I will eventually find a better, less hacky way, but this was super easy for now and works fine.

Reply:
Okay! I see you this is how your identifying live or local environment for the API. Thanks. If we ever change our domain name we would need to update this here.